### ISSUE TITLE
Use Travis CI for open source projects as a CI system

### ISSUE THREAD

**Issue by skrzyneckik** [frogermcs/AndroidDevMetrics#48]
#46 added a few tests, so maybe it would good idea to run them with every commit/pr. @frogermcs what do you think?

**Comment by frogermcs** [comment]
Yes, definitely! It would help a lot with a contribution - esp. reviewing the code.
